models/folder.go

```diff
@@ -15,7 +15,7 @@ type Folder struct {
 	gorm.Model
 	Name     string `gorm:"unique_index:idx_only_one_name"`
 	ParentID *uint  `gorm:"index:parent_id;unique_index:idx_only_one_name"`
-	OwnerID  uint   `gorm:"index:owner_id"`
+	OwnerID  int    `gorm:"index:owner_id"`
```

Comment:
Existing operations on folders will break on group folders. E.g. query folder props, renaming folder.

Reply:
Create share link on group folder does not work.

Reply:
Move/Copy inside of a group folder, path selector does not work.
"Uploader" is shown as blank in the admin dashboard.